source/software/using_software.rst

```diff
@@ -1,41 +1,35 @@
 Using software
 ==============
 
-The best way to get a complete list of all available software in a
-particular cluster can be obtained by typing:
-
-::
-
-   $ module av
-
-In order to use the software stack in the HPC cluster, the user should work
-with the :ref:`module system <module system basics>`.
+A lot of scientific software is centrally available on the VSC clusters. To
+avoid conflicts between different software packages, the installations are
+offered as :ref:`modules <module_system_basics>`. The executables, libraries,
+headers, ... of a certain module can only be used after that module has been
+loaded. By loading a certain set of modules, you can easily set up an
+environment that has precisely the software you need.
+
+:ref:`Toolchains <toolchains>` are an important concept in this context.
+A toolchain consists of a set of compilers, MPI library and
+basic libraries that work together well with each other, and then a
+number of applications and other libraries compiled with that set of
+tools and thus often dependent on those. We use toolchains based on the
+Intel and GNU compilers, and refresh them twice a year, leading to
```

Comment:
at VUB we refresh them only once a year: 2022a, 2023a, ...

Reply:
(addressed in 0342428cfebcad6c63fb6b595acd610a27b61b1f)